src/devices/Ssd13xx/Commands/Ssd1327Commands/SetInternalVddRegulator.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ssd13xx.Commands.Ssd1327Commands
+{
+    public class SetInternalVddRegulator : ISsd1327Command
+    {
+        /// <summary>
+        /// This command is used to enable internal VDD regulator. 
+        /// </summary>
+        public SetInternalVddRegulator()
```

Comment:
should this take bool or have property `Enabled` which the byte depends on?

Reply:
Done